libs/utils/utils.js

```diff
@@ -448,6 +448,7 @@ export async function loadBlock(block) {
   }
 
   const name = block.classList[0];
+  const hasStyles = AUTO_BLOCKS.find((ab) => Object.keys(ab).includes(name))?.styles ?? true;
```

Comment:
I think we can make this shorter
```suggestion
  const hasStyles = !!AUTO_BLOCKS.find((ab) => Object.keys(ab).includes(name))?.styles;
```

Reply:
Or you can just leave it like this: 
`const hasStyles = AUTO_BLOCKS.find((ab) => Object.keys(ab).includes(name))?.styles`
The logic will still work

Reply:
We don't define a `styles` property for auto blocks that _do_ need them. So `styles` would be `undefined` and `!!undefined` will be `false`, when in fact we do want styles to be loaded unless explicitly stated otherwise.

Reply:
Got it 👍